<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvoiceQueryInputWidget</class>
 <widget class="QWidget" name="InvoiceQueryInputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="2">
    <widget class="QSpinBox" name="daysPrevSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QSpinBox" name="importIntervalSpinBox">
     <property name="minimum">
      <number>10000</number>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QSpinBox" name="chunkSizeSpinBox">
     <property name="minimum">
      <number>1000</number>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="daysPrevLabel">
     <property name="text">
      <string>How many days of records to retrieve</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="chunkSizeLabel">
     <property name="text">
      <string>Chunk Size</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QTimeEdit" name="importTimeEdit">
     <property name="displayFormat">
      <string>hh:mm</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="importIntervalCheckBox">
     <property name="text">
      <string>Import on millisecond intervals</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="importDailyCheckBox">
     <property name="text">
      <string>Import daily at this time</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="commitSettingsButton">
     <property name="text">
      <string>Commit Settings</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="loadExistingSettingsButton">
     <property name="text">
      <string>Load Existing Settings</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>loadExistingSettingsButton</tabstop>
  <tabstop>chunkSizeSpinBox</tabstop>
  <tabstop>daysPrevSpinBox</tabstop>
  <tabstop>importIntervalCheckBox</tabstop>
  <tabstop>importDailyCheckBox</tabstop>
  <tabstop>importIntervalSpinBox</tabstop>
  <tabstop>importTimeEdit</tabstop>
  <tabstop>commitSettingsButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
